app/src/main/java/org/sopt/and/screen/LogInScreen.kt

```diff
@@ -0,0 +1,303 @@
+            Text(
+                text = "Wavve",
+                color = colorResource(R.color.white),
+                fontSize = 30.sp,
+                fontWeight = FontWeight.Bold
+            )
+
+            Spacer(Modifier.padding(40.dp))
+
+            TextField_Custom2(
+                id,
+                { text: String -> id = text },
+                "이메일 주소 또는 아이디",
+                true,
+                keyboardOptions = KeyboardOptions(
+                    imeAction = ImeAction.Next,
+                )
+            )
+
+            Spacer(Modifier.padding(5.dp))
+
+            TextField_Custom2(
+                password,
+                { text: String -> password = text },
+                "비밀번호",
+                showed = showPassword,
+                keyboardOptions = KeyboardOptions(
+                    imeAction = ImeAction.Done,
+                )
+            ) {
+                TextButton(
+                    onClick = {
+                        showPassword = !showPassword
+                    },
+                    content = {
+                        Text(
+                            text = "show", modifier = modifier.padding(7.dp)
+                        )
+                    }
+                )
+            }
+
+
+            Column(
+                horizontalAlignment = Alignment.CenterHorizontally,
+                modifier = Modifier.padding(top = 30.dp)
+            ) {
+                Button(
+                    onClick = {
+                        if (signUpId.isNotEmpty()) {
+                            if (id != signUpId || password != signUpPassword) {
+                                scope.launch {
+                                    val result = snackbarHostState
+                                        .showSnackbar(
+                                            message = "아이디와 비밀번호를 확인해주세요",
+                                            actionLabel = "확인하기",
+                                            duration = SnackbarDuration.Short
+                                        )
+                                    when (result) {
+                                        SnackbarResult.ActionPerformed -> {
+                                            Toast.makeText(
+                                                context,
+                                                "id :$signUpId\npassword :$signUpPassword",
+                                                Toast.LENGTH_SHORT
+                                            ).show()
+                                        }
+                                        SnackbarResult.Dismissed -> {
+                                        }
+                                    }
+                                }
+                            } else {
+                                scope.launch {
+                                    Toast.makeText(context, "로그인 하였습니다", Toast.LENGTH_SHORT).show()
+                                }
```

Comment:
```suggestion
                                Toast.makeText(context, "로그인 하였습니다", Toast.LENGTH_SHORT).show()
```

Toast.makeText는 suspend 함수가 아니기 때문에 일반함수 호출하듯이 사용가능합니다.

Reply:
넵 구별 잘해보겠습니다!